### PR TITLE
feat: clear cache button

### PR DESCRIPTION
**Pull request by 0xdef1cafe**
## Description

requested by ops - a button in the settings modal to clear the redux cache and reload the app

this PR will

* clear the redux cache
* navigate the user back to the dashboard
* do a hard reload of the app

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

n/a

## Risk

low - clearing cache is useful for support to triage issues

## Testing

* hit the button
* notice the app reloads
* see wallet/earn/rewards balance repopulate
* see consent banner reappears (this won't mess with mixpanel tracking)

### Engineering

as above

### Operations

as above

## Screenshots (if applicable)

https://user-images.githubusercontent.com/88504456/228676635-92b5469a-d1ef-43ff-a466-cb0753a6466f.mov

### CODE REVIEW

File: src/components/Modals/Settings/SettingsList.tsx [hunk 1]
```diff
@@ -87,6 +87,17 @@ export const SettingsList: FC<SettingsListProps> = ({ appHistory }) => {
     }
   }
 
+  const handleClearCacheClick = async () => {
```
Comment (gomesalexandre) [inline src/components/Modals/Settings/SettingsList.tsx:90]: `useCallback()` if we feel like it

**Review by gomesalexandre — APPROVED** [review]
LGTM, confirmed this does what it says on the box

File: src/components/Modals/Settings/SettingsList.tsx (hunk 1 above)
Comment (0xdef1cafe) [reply src/components/Modals/Settings/SettingsList.tsx:90]: good call, incoming

Comment (0xdef1cafe) [reply src/components/Modals/Settings/SettingsList.tsx]: https://github.com/shapeshift/web/pull/4155/commits/f44c18c9da95dad8f297f3fdea2528378e4ae22d

**Review by gomesalexandre — APPROVED** [review]
trampoline

**Pull request merged**